www/modules/recall/views/Index.scss

```diff
@@ -1,7 +1,13 @@
 .recall.index {
     md-content {
         background: lighten(lightgrey, 10%);
+
+        > div {
```

Comment:
We should probably make this a mixin, or apply globally at app level for all md-contents?